api/src/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# node:18.12.1-alpine3.17
+#FROM node@sha256:b375b98d1dcd56f5783efdd80a4d6ff5a0d6f3ce7921ec99c17851db6cba2a93
```

Comment:
@joseph-bisonai How about we start using the alpine?

Reply:
I tried that alpine but it crashed.